app/components/theme/switcher_component.rb

```diff
@@ -1,25 +1,28 @@
 class Theme::SwitcherComponent < ViewComponent::Base
-  def initialize(theme:, mobile: false, icon_only: false)
-    @theme = theme
-    @mobile = mobile
-    @icon_only = icon_only
+  include Classy::Yaml::ComponentHelpers
```

Comment:
This is a neat idea. Always seemed a bit cumbersome to pass class names into a component